dts/bindings/spi/atmel,sam-spi.yaml

```diff
@@ -18,3 +18,16 @@ properties:
       type: int
       description: peripheral ID
       required: true
+
+    pinctrl-0:
+      type: phandles
+      required: true
+      description: |
+        PIO pin configuration for the various SPI signals that includes
+        MISO, MOSI, SPCK, and possibly various chipselects signals.  We
```

Comment:
need space between **chip** and **select**.

Reply:
done